app/components/resource_button.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+class ResourceButton < ApplicationComponent
+  attr_reader :resource, :policy
+
+  def initialize(resource:, policy: nil)
+    @resource = resource
+    @policy = policy
```

Comment:
@nicktiberi we don't appear to be using policy anywhere. I recall that we were at one point. Do we need to?

Reply:
Nm... I just wasn't seeing it.